src/grid/types/formio-submission.ts

```diff
@@ -0,0 +1,17 @@
+export default interface FormioSubmission<T = any, stateType= state> {
+  _id?: string;
+  created?: string;
+  data?: T;
+  form?: string;
+  modified?: string;
+  owner?: string;
+  project?: string;
+  state?: stateType;
+  _fvid?: number;
+  _vid?: number;
+}
+
+enum state {
```

Comment:
@vlad-shusterman 
1. could you please provide more specific name for this enum? Like `FormioSubmissionState` or something like that. 
2. Could you also please use make first capital letter for the name of enum?
3. Please make enum exportable so that it can be used outside ot this file